CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## master / unreleased
+* [BUGFIX] ThanosEngine: Only enable default optimizers. #6776
```

Comment:
We need to sort changelog items using predefined order mentioned in https://github.com/cortexproject/cortex/blob/master/RELEASE.md#prepare-your-release.

Since there are other items that are not ordered, we can skip for now and fix it at release time